demos/quickstart/cdn_loader/index.html

```diff
@@ -0,0 +1,63 @@
+﻿<!DOCTYPE html>
+<html>
+
+<head>
+    <title>AMD + UNPKG</title>
+    <meta charset="utf-8" />
+    <style>
+        body {
+            padding: 0px;
+            margin: 0px;
+            overflow: hidden;
+        }
+
+        #placeholder {
+            position: absolute;
+            top: 8px;
+            bottom: 8px;
+            left: 8px;
+            right: 8px;
+        }
+    </style>
```

Comment:
If the #placeholder style uses `position: fixed` instead of `position: absolute` (and give it `overflow: hidden`) then the `body` style isn't needed.

Reply:
Looks like the body style isn't needed in either case...  Which is better "fixed" or "absolute"?

Reply:
In this case "fixed" because "absolute" is supposed to have an ancestor element with `position: relative`